firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -1,303 +1,298 @@
 /*
  * Brave firmware state machine for single Boron
  * written by Heidi Fedorak, Apr 2021
- * 
+ *
  *  Yes, I straight up copied and pasted a function four times :-P
- *  Reason 1: it makes the console functions easier to use, user just has to 
- *             input an int to the correct function, and doesn't have to 
+ *  Reason 1: it makes the console functions easier to use, user just has to
+ *             input an int to the correct function, and doesn't have to
  *             remember/type correctly a string command.
- *  Reason 2: I don't have to spend time programming a command parser for 
+ *  Reason 2: I don't have to spend time programming a command parser for
  *              things like 60 -threshold vs 100 -stillnessTime
  *
  *
  *
-*/
-#include "Particle.h"
+ */
 #include "consoleFunctions.h"
+
+#include "Particle.h"
 #include "flashAddresses.h"
-#include "stateMachine.h"
 #include "imDoorSensor.h"
+#include "stateMachine.h"
 
-void setupConsoleFunctions(){
-
-  //particle console function declarations, belongs in setup() as per docs
-  Particle.function("Force_Reset", force_reset); 
-  Particle.function("Turn_Debugging_Publishes_On_Off", toggle_debugging_publishes);   
-  Particle.function("Change_Initial_Timer", initial_timer_set); 
-  Particle.function("Change_Duration_Timer", duration_timer_set);     
-  Particle.function("Change_Stillness_Timer", stillness_timer_set);  
-  Particle.function("Change_INS_Threshold", ins_threshold_set);
-  Particle.function("Change_IM21_Door_ID", im21_door_id_set); 
+void setupConsoleFunctions() {
+    // particle console function declarations, belongs in setup() as per docs
+    Particle.function("Force_Reset", force_reset);
+    Particle.function("Turn_Debugging_Publishes_On_Off",
+                      toggle_debugging_publishes);
```

Comment:
It's strange that it did this, because I actually did not set a column limit. Do you think 150 would still be a good value to use with the increased indent spacing?

Reply:
Yup, I think 150 is still a good idea. Thanks!